science/lscsoft-deps/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem      1.0
 
 name            lscsoft-deps
 version         20190213
+revision        1
```

Comment:
No need to bump the revision for this port, it's just a meta-port that makes sure a given set of ports is installed.